tests/MongoDB.Driver.Tests/Specifications/command-monitoring/TestRunner.cs

```diff
@@ -93,6 +92,7 @@ public bool OneTimeSetup()
                     cb.ConfigureServer(ss => ss.With(heartbeatInterval: Timeout.InfiniteTimeSpan));
                 }
             };
+            settings.RetryWrites = false;
```

Comment:
Otherwise, this test fails with the following error:
```Expected object to be {
	delete  = test,
	deletes = [{
			"q": {
				"_id": {
					"$nothing": 1
				}
			},
			"limit": 1
		}
	],
	ordered = true,
	$db = command - monitoring - tests
}, but found {
	delete  = test,
	ordered = true,
	txnNumber = 28,
	$db = command - monitoring - tests,
	deletes = [{
			"q": {
				"_id": {
					"$nothing": 1
				}
			},
			"limit": 1
		}
	]
}
```

Reply:
This doesn't feel right. This configuration affects ALL command monitoring tests.

We should let RetryWrites have its default value and fix the small number of tests that might require overridding the new default value. In fact, when an override is necessary for the test to pass shouldn't that override be part of the JSON test itself?

Reply:
OK. Since we are not updating these JSON tests (yet) we can set RetryWrites to false.